chat-plugins/animals.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+const http = require('http');
+const AnimalsHelp = '|raw|<div class="infobox"><strong>Animals Plugin by <font color="' + Gold.hashColor(color) + '">DarkNightSkies</font> & <font color="' + Gold.hashColor('kyvn') + '">Kyv.n(♥)</font><strong><br><ul><li>/animals cat - Displays a cat</li><li>/animals kitten - Displays a kitten</li><li>/animals dog - Displays a doge</li><li>/animals puppy - Displays a puppy</li><li>/animals bunny - Displays a bunny</li><li>/animals otter - Displays an otter</li><li>/animals pokemon - Displays a pokemon</li></ul></div>';
```

Comment:
this isn't how we do help commands

At the bottom of the command do like, ``animalhelp: ['What you want it to say'],``